### PR TITLE
week1 homework ahmet yasin

### PR DESCRIPTION
**Pull request by ahmtysn**
I did this homework last week and i didn't know that i should do commit after every exercise. I'm sorry..Just I have a few commit..

### CODE REVIEW

File: Week1/homework/js-exercises/10_compare.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+"use strict";
+const cities = ["paris", "london", "madrid", "amsterdam"];
+const names = ["john", "wouter", "unmesh", "martin", "noer", "ahmet", "yasin"];
+console.log(`cities variable have ${cities.length} elements..`);
```
Comment (wilgert) [inline Week1/homework/js-exercises/10_compare.js]: I'm just nitpicking here but correct English would be "cities variable **has**"

File: Week1/homework/js-exercises/2_error.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+"use strict";
+/* console.log('I'm awesome');
+SyntaxError: missing ) after argument list
+We have two options to fix problem..*/
+
+console.log("I'm awesome");
+console.log('I\'m awesome');
```
Comment (wilgert) [inline Week1/homework/js-exercises/2_error.js:7]: very good!

File: Week1/homework/js-exercises/6_array.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+"use strict";
+let items = [];
+console.log("Empty Array:");
+console.log(items);
+let animals = ["lion", "tiger", "jaguar"];
+console.log(`${animals[1]} is the second element..`);
```
Comment (wilgert) [inline Week1/homework/js-exercises/6_array.js]: the instructions asked to log the second array, not the second item in te array.

**Review by wilgert — CHANGES_REQUESTED** [review]
You did really well, only some small things are missing! Please fix these and I will aprove the homework!

**Comment by ahmtysn** [comment]
i think i fixed according to your advice, thanks for your interest @wilgert

**Review by wilgert — APPROVED** [review]

**Pull request closed without merging**